narwhal/primary/src/primary.rs

```diff
@@ -157,11 +147,6 @@ impl Primary {
             &primary_channel_metrics.tx_certificate_fetcher,
             &primary_channel_metrics.tx_certificate_fetcher_total,
         );
-        let (tx_block_synchronizer_commands, rx_block_synchronizer_commands) = channel_with_total(
-            CHANNEL_CAPACITY,
-            &primary_channel_metrics.tx_block_synchronizer_commands,
```

Comment:
Let's also remove the metrics from here https://github.com/MystenLabs/sui/blob/c767d31e180ee2ebb5e722a9ea821ad374a21716/narwhal/primary/src/metrics.rs#L74C9-L74C39